docker/beta/uwsgi/uwsgi.service

```diff
@@ -1,25 +1,57 @@
 #!/bin/bash
 
+function is_cron_up {
+    if [ pid = $(pgrep cron) ]
+    then
+        return 1
+    else
+        return 0
+    fi
+}
+
+function bring_cron_down {
+    if [ is_cron_up ]
+    then
+        service cron stop
+    fi
+}
+
+function bring_cron_up {
+    if ! [ is_cron_up ]
+    then
+        exec cron -f
+    fi
+}
+
 if [ "${CONTAINER_NAME}" = "listenbrainz-web-beta" ]
 then
+    bring_cron_down
     exec uwsgi /etc/uwsgi/uwsgi.ini
 fi
 
 if [ "${CONTAINER_NAME}" = "listenbrainz-influx-writer-beta" ]
 then
+    bring_cron_down
```

Comment:
I don't fully understand why you bring cron down in all containers. Do you see cron starting automatically in all other containers when you bring them up?
From what I understand cron won't start automatically in all containers when you start from docker, only the specified command will execute. In this case I think it makes sense to run `cron -f` in the scheduler container, but not stop it in the others.